### DISCUSSION TITLE
Sample or getting started template for MAUI Blazor

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#1152] [Q&A]
Hi,
Do you know of any sample or getting started template for a MAUI Blazor App using fluent ui. Thanks.

**Comment by vnbaaij** [comment]
As far asI know, nothing is available. What you could do is start with one of our regula( Web App or WebAssembly) templates, then create a project with the Hybrid tmplate and copy over files and code from the Fluent app to the Hybrid app.
